Whitebeet.py

```diff
@@ -1727,11 +1733,17 @@ def v2gEvseParsePaymentSelected(self, data):
         message = {}
         self.payloadReaderInitialize(data, len(data))
         message['selected_payment_method'] = self.payloadReaderReadInt(1)
+
+        print("payment option:")
+        print(message['selected_payment_method'])
         if message['selected_payment_method'] == 1:
             message['contract_certificate'] = self.payloadReaderReadBytes(self.payloadReaderReadInt(1))
             message['mo_sub_ca1'] = self.payloadReaderReadBytes(self.payloadReaderReadInt(1))
             message['mo_sub_ca2'] = self.payloadReaderReadBytes(self.payloadReaderReadInt(1))
             message['emaid'] = self.payloadReaderReadBytes(self.payloadReaderReadInt(1)).decode('utf-8')
+
+        self.payloadBytesRead = self.payloadBytesLen

```

Comment:
Why is this needed?

Reply:
To set the number of read bytes to the number of bytes of the message.